site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,173 @@
++++
+date = "2020-06-07T03:15:00-03:00"
+title = "Progress Report May 2020"
+author = "GoldenX86"
+forum = 0
++++ 
+
+Hello yuz-ers! How are you all doing?
+
+In this monthly episode of "yuzu — Trials and Tribulations," we offer you: major rewrites, massive performance gains, 
+stability improvements, bug fixes and graphical corrections.
+*More after the commercial break.*
+<!--more-->
+
+## The worst kept secret
+
+It has its own [article](https://yuzu-emu.org/entry/yuzu-prometheus/), and it had been guessed to hell and back before the official announcement.
+`Project Prometheus` is a proper multithreaded emulation of the 4 CPU cores the Nintendo Switch offers.
+This brings a massive performance boost to users with CPUs with 4 physical cores or more, but for this to happen, a lot of groundwork was needed.
+Besides changes previously discussed in past reports, old external libraries (which yuzu needs to operate) needed to be updated,
+and with that, some changes were needed for our Linux users. 
+
+Thanks to [jroweboy’s](https://github.com/jroweboy) work, yuzu now uses [Conan](https://conan.io/),
+helping the project [manage dependencies](https://github.com/yuzu-emu/yuzu/pull/3735), and letting Linux distributions use their native ones when possible.
+
+With the previous VMM rewrite reducing memory use, the dependencies updated, and all the groundwork done, [Blinkhawk](https://github.com/FernandoS27) 
+pressed the metaphorical nuclear launch button and [released Project Prometheus](https://github.com/yuzu-emu/yuzu/pull/3955).
+yuzu now use up to 6 to 7 CPU threads (in ideal conditions) 
+compared to the previous 2 to 3. You should expect a performance boost in a lot of games, but still see some titles perform mostly the same due to being coded to only use a single thread.
+
+Now, some clarifications are needed for this change. Multicore support can’t be merged into our [Mainline](https://github.com/yuzu-emu/yuzu-mainline) release for now due to incompatibilities between Multicore and the [Master](https://github.com/yuzu-emu/yuzu) branch of yuzu. Work is being done to resolve the conflicts, but please have patience.
```

Comment:
[Master] branch of yuzu
Uhhhhhhhh... 

https://www.cnet.com/news/microsofts-github-is-removing-coding-terms-like-master-and-slave/

Reply:
MEH